tests/VCR/Storage/AbstractStorageTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace VCR\Storage;
+
+use org\bovigo\vfs\vfsStream;
+
+/**
+ * Test integration of PHPVCR with PHPUnit.
+ */
+class AbstractStorageTest extends \PHPUnit_Framework_TestCase
+{
+    protected $handle;
+    protected $filePath;
+    protected $storage;
+
+    public function testFilePathCreated()
+    {
+        $fs = vfsStream::setup('test');
+
+        $this->storage = new TestStorage(vfsStream::url('test/'), 'file');
+        $this->assertTrue($fs->hasChild('file'));
+
+        $this->storage = new TestStorage(vfsStream::url('test/'), 'folder/file');
```

Comment:
this line is failing. do you see what i do wrong when creating the folder? it looks like i do the right thing and mkdir on vfsStream is not working properly... which seems odd.